### ISSUE TITLE
Packaging issues

### ISSUE THREAD

**Issue by echeung-amzn** [cdklabs/cdk-monitoring-constructs#18]
Tracking some issues with the npm tarball:

- [x] TS files are included
- [x] Lambda assets aren't transpiled

**Comment by voho** [comment]
I am not 100% sure, but I think it could be these config differences in tsconfig:

Desired setting:

```
  "include": [
    "**/*.ts"
  ],
  "exclude": [
    "node_modules",
    "dist/**/*.ts"
  ],
```

Current setting:

```
  "include": [
    "lib/**/*.ts"
  ],
  "exclude": [
    "node_modules"
  ],
```

**Comment by echeung-amzn** [comment]
Addressed by #23 and #24.
